motor/core.py

```diff
@@ -1535,6 +1537,9 @@ class _LatentCursor(object):
     _CommandCursor__killed = False
     cursor_id = None
 
+    def _CommandCursor__end_session(self, *args, **kwargs):
```

Comment:
This was required to fix this error:
```
ERROR: test_cancelled_error_fetch_next_aggregate (test.asyncio_tests.test_asyncio_cursor.TestAsyncIOCursor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/shane/git/motor/test/asyncio_tests/__init__.py", line 48, in __call__
    result = self.orig_method()
  File "/Users/shane/git/motor/test/test_environment.py", line 283, in wrap
    return f(*args, **kwargs)
  File "/Users/shane/git/motor/test/asyncio_tests/__init__.py", line 217, in wrapped
    raise coro_exc from None
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/shane/git/mongo-python-driver/pymongo/command_cursor.py", line 87, in __die
    print(self.__address)
AttributeError: '_LatentCursor' object has no attribute '_CommandCursor__address'
```

Reply:
Sorry, what is this failure? I can't see this print statement in the code. I can see that `__die` calls `_CommandCursor__end_session` so it makes sense that you needed this method but I can't figure out what this traceback is telling us.

Reply:
Updated to include the correct error traceback. The earlier print-statement one was from debugging. I also fixed the `KeyError: 'exception'` issue. The context's exception field is optional: https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.call_exception_handler